codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/typescript/client/files_producers/EntryPointFileProducer.kt

```diff
@@ -0,0 +1,49 @@
+package io.vrap.codegen.languages.typescript.client.files_producers
+
+import com.google.inject.Inject
+import com.google.inject.name.Named
+import io.vrap.codegen.languages.typescript.client.AbstractRequestBuilder
+import io.vrap.rmf.codegen.di.VrapConstants
+import io.vrap.rmf.codegen.io.TemplateFile
+import io.vrap.rmf.codegen.rendring.FileProducer
+import io.vrap.rmf.codegen.rendring.utils.keepIndentation
+import io.vrap.rmf.codegen.types.VrapTypeProvider
+import io.vrap.rmf.raml.model.modules.Api
+import io.vrap.rmf.raml.model.resources.Resource
+
+class EntryPointFileProducer @Inject constructor(
```

Comment:
May be we should name this `ApiRootFileProducer`?

Reply:
good idea